File: java/src/main/java/org/mydotey/scf/facade/LabeledStringProperties.java
package org.mydotey.scf.facade;

import org.mydotey.scf.labeled.LabeledConfigurationManager;
import org.mydotey.scf.labeled.LabeledKey;

/**
 * @author koqizhao
 *
 * May 21, 2018
 */
public class LabeledStringProperties extends StringValueProperties<LabeledKey<String>, LabeledConfigurationManager> {

    public LabeledStringProperties(LabeledConfigurationManager manager) {
        super(manager);
    }

}

File: dotnet/src/MyDotey.SCF.Labeled/Facade/LabeledStringProperties.cs
using System;

using MyDotey.SCF.Labeled;

namespace MyDotey.SCF.Facade
{
    /**
     * @author koqizhao
     *
     * May 21, 2018
     */
    public class LabeledStringProperties : StringValueProperties<LabeledKey<String>, ILabeledConfigurationManager>
    {
        public LabeledStringProperties(ILabeledConfigurationManager manager)
            : base(manager)
        {

        }
    }
}